apps/ehr/src/pages/reports/DailyPayments.tsx

```diff
@@ -113,6 +115,35 @@ export default function DailyPayments(): React.ReactElement {
     }
   }, []);
 
+  /**
+   * Splits a date range into batches of maximum BATCH_DAYS days each
+   */
+  const splitDateRangeIntoBatches = useCallback(
```

Comment:
if it's the same function as [the previous one](https://github.com/masslight/ottehr/pull/4831/files#diff-43931300144d6ae81b9b70a57a699175af9fcf73d43828e390e48915254038c9R83), why not make it a constant? then you can pass the arrow function into `useCallback` and still get it memoized.